content/about/resources.fr.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Resources
```

Comment:
Just a quick typo: title should be `resources` and not `Resources`, I know it's minimal, but it's just to guarantee, your file metadata matches the english one!